ext/curl/multi.c

```diff
@@ -60,12 +60,17 @@ static inline php_curlm *curl_multi_from_obj(zend_object *obj) {
 PHP_FUNCTION(curl_multi_init)
 {
 	php_curlm *mh;
+	CURLM *multi;
 
 	ZEND_PARSE_PARAMETERS_NONE();
-
+	multi = curl_multi_init();
+	if(UNEXPECTED(multi == NULL)) {
+		zend_throw_error(NULL, "Could not initialize a new cURL multi handle");
```

Comment:
Not sure if this should be an error or exception, looking at interface.c these kinds of failures throw an exception with a wording like "Failed to ..." (see interface.c)
Anyway, I'll let @adoy have the final say as he maintains ext/curl.

Reply:
made the error message more interface.c-like